config/core/configmaps/default-pingsource.yaml

```diff
@@ -23,5 +23,5 @@ metadata:
     knative.dev/example-checksum: "6eaeecba"
 data:
   ping-config: |
-    # dataMaxSize: 10  # Max number of bytes allowed to be sent for message excluding any base64 decoding.
```

Comment:
if `-1` is the default and corresponds to no limit, why not have an integer field that isn't specified by default and only has an `_example` entry?

Reply:
This was modeled after https://github.com/knative/eventing/blob/master/config/core/configmaps/default-channel.yaml 
Why should this be any different here ?

Reply:
`default-ch-config` takes an entire YAML object as a value, which is harder to default since you need to make assumptions on what types of channel are available in the cluster.

For this feature, we are talking about simple keys/values, so the pattern below sounds much more suitable to me:

```yaml
data:
  _example: |
    ################################
    #                              #
    #    EXAMPLE CONFIGURATION     #
    #                              #
    ################################

    # Max number of bytes allowed to be sent
    dataMaxSize: -1

  dataMaxSize: 4096
```

The `_example` key is just a nice to have, but it's useful for users if the ConfigMap is deployed by default.

Reply:
@antoineco  thx for that explanation.  Due to other work I probably at this time won't get to make these changes before this coming release